tests/integration/telemetry/test_experimental_metric.py

```diff
@@ -51,7 +52,11 @@ def test_must_send_experimental_metrics_if_experimental_command(self):
                             "debugFlagProvided": ANY,
                             "region": ANY,
                             "commandName": ANY,
-                            "metricSpecificAttributes": {"experimentalAccelerate": True, "experimentalAll": False},
+                            "metricSpecificAttributes": {
+                                "projectType": "CFN",
```

Comment:
I think there is no template used for `sam traces` command, so I think we should not use project type here.

Reply:
Updated the metric adding logic, skip adding project type if the template is not provided.